mkdocs/overrides/devbook/transactions/messages.mjs

```diff
@@ -0,0 +1,164 @@
+	'0000000000000000000000000000000000000000000000000000000000000000';
+const senderKeyPair = new SymbolFacade.KeyPair(
+	new PrivateKey(senderPrivateKey));
+const senderAddress = facade.network.publicKeyToAddress(
+	senderKeyPair.publicKey);
+
+// Set up recipient (using public key only)
+const recipientPublicKeyString = process.env.RECIPIENT_PUBLIC_KEY ||
+	'D04AB232742BB4AB3A1368BD4615E4E6D0224AB71A016BAF8520A332C9778737';
+
+const recipientPublicKey = new PublicKey(recipientPublicKeyString);
+const recipientAddress = facade.network.publicKeyToAddress(
+	recipientPublicKey);
+console.log('Sender address:', senderAddress.toString());
+console.log('Recipient address:', recipientAddress.toString(), '\n');
+
+// ===== PLAIN TEXT MESSAGE =====
+console.log('=== Sending Plain Text Message ===');
+
+// Fetch current network time
+const timePath = '/node/time';
+console.log('Fetching current network time from', timePath);
+const timeResponse = await fetch(`${NODE_URL}${timePath}`);
+const timeJSON = await timeResponse.json();
+let timestamp = new NetworkTimestamp(
+	timeJSON.communicationTimestamps.receiveTimestamp);
+console.log('  Network time:', timestamp.timestamp,
+	'ms since nemesis');
+
+// Fetch recommended fees
+const feePath = '/network/fees/transaction';
+console.log('Fetching recommended fees from', feePath);
+const feeResponse = await fetch(`${NODE_URL}${feePath}`);
+const feeJSON = await feeResponse.json();
+const medianMult = feeJSON.medianFeeMultiplier;
+const minimumMult = feeJSON.minFeeMultiplier;
+const feeMult = Math.max(medianMult, minimumMult);
+console.log('  Fee multiplier:', feeMult);
+
+// Create a plain text message
+const plainMessage = new TextEncoder().encode('Hello, Symbol!');
+console.log('Plain message:',
+	new TextDecoder().decode(plainMessage));
+
+// Build transfer transaction with plain message
+let transaction = facade.transactionFactory.create({
+	type: 'transfer_transaction_v1',
+	signerPublicKey: senderKeyPair.publicKey.toString(),
+	deadline: timestamp.addHours(2).timestamp,
+	recipientAddress: recipientAddress.toString(),
+	mosaics: [{
+		mosaicId: generateMosaicAliasId('symbol.xym'),
+		amount: 1_000_000n	// 1 XYM
+	}],
+	message: plainMessage
+});
+transaction.fee = new models.Amount(feeMult * transaction.size);
+
+// Sign and announce the transaction
+let signature = facade.signTransaction(senderKeyPair, transaction);
+let jsonPayload = facade.transactionFactory.static.attachSignature(
+	transaction, signature);
+let transactionHash = facade.hashTransaction(transaction).toString();
+console.log('Transaction hash:', transactionHash);
+
+let announceResponse = await fetch(`${NODE_URL}/transactions`, {
+	method: 'PUT',
+	headers: { 'Content-Type': 'application/json' },
+	body: jsonPayload
+});
+console.log('Plain message transaction announced\n');
+
+// Wait a moment before sending the next transaction
+await new Promise(resolve => setTimeout(resolve, 2000));
```

Comment:
Why do you need to wait? This will make readers think that the wait is necessary.